urwid/tests/test_vterm.py

```diff
@@ -226,6 +240,7 @@ def test_erase_display(self):
         self.expect('   x5a98765')
 
     def test_scrolling_region_simple(self):
+        # TODO(Aleksei): Issue #544
```

Comment:
4 tests which fail on regular basis

Reply:
we can mark them as flaky or to be skipped so they don't fail unrelated changes, can't we?